choreographer/browser.py

```diff
@@ -55,33 +56,64 @@ def __init__(
         loop=None,
         executor=None,
         debug=False,
-        debug_browser=None,
+        debug_browser=False,
         **kwargs
     ):
         # Configuration
         self.enable_gpu = kwargs.pop("enable_gpu", False)
         self.enable_sandbox = kwargs.pop("enable_sandbox", False)
+        self._tmpdir_path = kwargs.pop("tmp_path", None)
         if len(kwargs):
             raise ValueError(f"Unknown keyword arguments: {kwargs}")
         self.headless = headless
         self.debug = debug
         self.loop_hack = False # subprocess needs weird stuff w/ SelectorEventLoop
 
         # Set up stderr
-        if not debug_browser:  # false o None
+        if debug_browser is False:  # false o None
             stderr = subprocess.DEVNULL
         elif debug_browser is True:
-            stderr = None
+            stderr = sys.stderr
         else:
             stderr = debug_browser
+
+        # awful
+        if stderr and stderr != subprocess.PIPE and stderr != subprocess.STDOUT and stderr != subprocess.DEVNULL and not isinstance(stderr, int):
```

Comment:
I think that this can be easy to read if you create some variables with some conditionals, like:
```
stuff_1= stderr != subprocess.PIPE
stuff_2 = stderr != subprocess.STDOUT
stuff_3 = stderr != subprocess.DEVNULL
if stderr and  stuff_1 and stuff_2 and stuff_3 and not isinstance(stderr, int):
```
or
```
stuff = [subprocess.PIPE, subprocess.STDOUT, subprocess.DEVNULL]
if stderr and  stderr not in stuff and not isinstance(stderr, int):
```